kotlin_client/src/main/java/com/github/paylike/kotlin_client/domain/payment/response/PaylikeClientResponse.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.paylike.kotlin_client.domain.payment.response
+
+import java.lang.Exception
+
+/**
+ * Describes the client response from the Paylike capture API
+ */
+class PaylikeClientResponse(
+    val paymentResponse: PaymentResponse,
+    val isHTML: Boolean,
+    htmlBody: String?,
+) {
+    /**
+     * Returns HTML body if not null
+     * otherwise throws an exception
+     */
+    val htmlBody: String? = htmlBody
+        get() {
```

Comment:
Does Kotlin now that you can only return `String` here which cannot be null (hence we have an exception for the other case)? Otherwise this function does not help 😄

Reply:
Done.